docs/modules/admin/pages/distributed_install.adoc

```diff
@@ -0,0 +1,136 @@
+
+:imagesdir: ../assets/images
+= Distributed Installation
+
+A distributed installation runs ALEC as a service on top of https://docs.opennms.com/horizon/latest/deployment/sentinel/introduction.html[Sentinel], an OpenNMS component that enables scalability for data processing.
+
+.Example distributed installation network architecture
+image::distributed_deployment.png[Network architecture graphic that displays the components of a distributed installation of ALEC, 700]
+
+[[prequisites]]
+== Prerequisites
+
+In addition to an existing instance of OpenNMS Horizon 30.0.1 or newer, these instructions assume that you have the following:
+
+* A Kafka broker (or cluster) running Kafka 0.11.x or greater.
+* One or more active Minions.
+* OpenNMS Sentinel.
+** Minions configured to use Kafka for the transport.
+** Minions ready to receive SNMP traps and Syslog messages.
+* ZooKeeper 3.5.10 or newer.
+
+== Before you begin
+
+You must complete the xref:install:pre_install.adoc[pre-installation tasks] before proceeding with the installation.
+
+Make sure that you install the package repository on all systems that are running Sentinel and OpenNMS.
```

Comment:
This might not belong in the "Before you begin" section... Perhaps it should be moved to the section on installing the packages?